weave-js/src/components/Panel2/ChildPanelExportReport/computeReportSlateNode.ts

```diff
@@ -4,7 +4,7 @@ import {ChildPanelFullConfig} from '../ChildPanel';
 import {getConfigForPath} from '../panelTree';
 import {toWeaveType} from '../toWeaveType';
 
-type WeavePanelSlateNode = {
+export type WeavePanelSlateNode = {
```

Comment:
Need to use this type in core app for a later change related to fixing the "clone node" issue. Figured I'd just throw it on there